tuto(serverless) - update tuto (#863)
scaleway/docs-content: tutorials/transform-images-using-faas-and-nodejs/index.mdx

  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
  - You have created a [Serverless Function namespace](/compute/functions/how-to/create-a-functions-namespace/)

>   - You have downloaded the corresponding [kubeconfig](/compute/kubernetes/how-to/edit-cluster/) file and [kubectl](/compute/kubernetes/how-to/connect-cluster-kubectl/) is configured and working

Reviewer: ligne supprimé

## Creating the BucketScan function

> 1. Set the following Environment Vairables at the function level:

Reviewer: ```suggestion
1. Set the following environment variables at the function level:
```

       srcKey = content.Key;
       const typeMatch = srcKey.match(/\.([^.]*)$/);
       if (!typeMatch) {
         console.log("Could not determine the image type.");
       } else {
         const imageType = typeMatch[1].toLowerCase();
         // Check that the image type is supported
         if (["jpeg", "jpg", "png"].includes(imageType) !== true) {
           return console.log(`Unsupported image type: ${imageType}`);
         } else {
           try {
             console.log(TRANSFORM_URL + "?key=" + srcKey);
             https.get(TRANSFORM_URL + "?key=" + srcKey);
             counter += 1;
           } catch (error) {
             console.log(error);
           }
         }
       }
     });
    ```
    * Return http call results:
    ```js 
              return {
            statusCode: 200,
            body: JSON.stringify({
              status: "Done",
              message: `${counter} images from the bucket have been processed over ${total} files in the bucket`,
            }),
            headers: {
              "Content-Type": "application/json",
            },
          };
        }
      });
    };
    ```

## Creating the ImageTransformation function

> The following steps are adapted from the [AWS lambda](https://docs.aws.amazon.com/lambda/latest/dg/with-s3-tutorial.html#with-s3-tutorial-create-function-code) tutorial. We recommand using the last version of Node.

Reviewer: ```suggestion
The following steps are adapted from the [AWS lambda](https://docs.aws.amazon.com/lambda/latest/dg/with-s3-tutorial.html#with-s3-tutorial-create-function-code) tutorial. We recommend using the latest version of Node for this tutorial.
```

1. Install `sharp` using the fbs `linuxmusl` platform to ensure compatibility with Scaleway’s node runtime:
    ```bash
    npm install --platform=linuxmusl --arch=x64 sharp --ignore-script=false --save
    ```

> 2. Set the following Environment Variables at the function level: 

Reviewer: ```suggestion
2. Set the following environment variables at the function level: 
```

            "Image : " +
            srcKey +
            " has successfully been resized and pushed to the bucket " +
            dstBucket
        }),
        headers: {
          "Content-Type": "application/json"
        }
      };
    };
    ```

## Pushing the two functions to Scaleway Serverless 

Please refer to the Serverless Documentation to learn how to package and deploy your functions using [Serverless Framework](/compute/functions/api-cli/fun-uploading-with-serverless-framework/) or a [zip-file](/compute/functions/how-to/package-function-in-zip/).

> Once the ImageTransform function is deployed copy its URL (from the Settings Tab in the console or in the Serverless Framework logs) into the BucketScan environment Varible. 

Reviewer: ```suggestion
Once the `ImageTransform` function is deployed, copy its URL (from the settings tab in the console or in the Serverless Framework logs) into the `BucketScan` environment variable. 
```

> Once everything is well deployed, test the BucketSan functions using its URL

Reviewer 2: ```suggestion
Once everything is deployed, test the BucketSan functions using its URL.
```
